src/services/rollouts.js

```diff
@@ -540,4 +543,216 @@ module.exports = class RolloutsHelper {
 			return error
 		}
 	}
+	/**
+	 * rollout publish
+	 * @method
+	 * @name publish
+	 * @param {Integer} rolloutId - rollout id
+	 * @returns {JSON} - rollout publish response.
+	 */
+
+	static async publish(rolloutId, orgId, loggedInUserId) {
+		try {
+			// fetch rollout details
+			const rolloutDetails = await this.details(rolloutId, orgId, loggedInUserId)
+
+			// check if rollout is present or not
+			if (rolloutDetails?.statusCode != 200 || rolloutDetails?.result == undefined) return rolloutDetails
+
+			if (rolloutDetails?.result?.status && rolloutDetails?.result?.status == common.ROLLOUT_PUBLISHED) {
+				return responses.failureResponse({
+					responseCode: 'CLIENT_ERROR',
```

Comment:
@adithyadinesh0412  responseCode will add by default in failureResponse so its no need to define here